firmware/defmt-test/macros/src/lib.rs

```diff
@@ -130,9 +131,14 @@ fn tests_impl(args: TokenStream, input: TokenStream) -> parse::Result<TokenStrea
                 }
             }
 
+            Item::Use(u) => {
+                imports.push(u);
+            }
+
             _ => {
                 return Err(parse::Error::new(
                     item.span(),
+                    // TODO Make error message generic over Item
                     "only function items are allowed",
```

Comment:
perhaps this should now say "only `#[test]` functions and imports (`use`) are allowed in this scope" (*). I think the error will have the right span and point to the item (e.g. a struct item) that's not allowed so I don't think it's super critical to report e.g. "this struct item is not allowed" from this error message.

(*) I *think* we don't allow non-`#[test]` functions here but can't remember

Reply:
> I think we don't allow non-#[test] functions here but can't remember

That is correct. 

This feature is only one addition to supported items. I did update the error to your suggestion. 
When more Items are supported the error should be made more generic, as manually declaring
which is, and which is not allowed seems error prone. 
However, this is nothing to worry about now. 

From my side this PR is ready to go :)